radp/digital_twin/mobility/tests/test_param_regression.py

```diff
@@ -0,0 +1,104 @@
+import unittest
+import numpy as np
+import pandas as pd
+
+from radp.digital_twin.mobility.param_regression import ParameterRegression
+
+
+class TestParameterRegression(unittest.TestCase):
+    def setUp(self) -> None:
+        """
+        Set up a mock DataFrame for testing the ParameterRegression class.
+
+        Initializes a DataFrame with random 'mock_ue_id' and 'velocity' values,
+        and creates an instance of ParameterRegression using this DataFrame.
+
+        How to Run:
+        ------------
+        To run these tests, execute the following command in your terminal:
+        ```
+        python3 -m unittest radp/digital_twin/mobility/tests/test_parameter_regression.py
+        ```
+        """
+        data = {
+            "mock_ue_id": np.random.randint(0, 10, size=100),
+            "velocity": np.random.uniform(0, 100, size=100),
+        }
```

Comment:
let's make this a fixed dataset -- choose just 2 UEs, with just 5 track points each